### PR TITLE
Create contribution-root.txt

### PR DESCRIPTION
**Pull request by deeptikohli**

### CODE REVIEW

File: contribution-root.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+contribution - project root (default reviewer)
```
Comment (brainupgrade-in) [inline contribution-root.txt:1]: please review this code  block and add more checks

**Review by brainupgrade-in — CHANGES_REQUESTED** [review]
add more checks

File: contribution-root.txt (hunk 1 above)
Comment (deeptikohli) [reply contribution-root.txt:1]: done

**Review by deeptikohli — COMMENTED** [review]
done

**Review by brainupgrade-in — APPROVED** [review]
verified checks.  change is approved

**Pull request merged**